src/components/Form/Form.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+import usersFromServer from '../../api/users';
```

Comment:
The `usersFromServer` data should be prepared in the `App` component and passed to the `Form` component as props. Avoid importing it directly in non-root components as per the checklist requirement: "Prepare data in one place (App component) and pass it to child components."